src/CMakeLists.txt

```diff
@@ -300,7 +300,7 @@ add_subdirectory(json_spirit)
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/xxHash")
 include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/rapidjson/include")
 
-find_package(fmt 5.2.1 QUIET)
+find_package(fmt 7.0.0 QUIET)
```

Comment:
how about lower the required version to 5.3.0 or 6.0.0. see https://github.com/fmtlib/fmt/blob/master/ChangeLog.rst#530---2018-12-28

as 5.3.0 is the first version in which fmt/chrono.h was officially introduced. and in 6.0.0 

> Removed deprecated fmt/time.h. Use fmt/chrono.h instead.

see https://github.com/fmtlib/fmt/blob/master/ChangeLog.rst#600---2019-08-26

EPEL8 only has 6.2.1 packaged at this moment. see https://centos.pkgs.org/8/epel-x86_64/fmt-6.2.1-1.el8.x86_64.rpm.html. and debian/unstable only has 6.1.2, see https://packages.debian.org/sid/libfmt-dev .

the most recent version packaged by ubuntu is 6.1.2 as well, see https://packages.ubuntu.com/search?keywords=libfmt-dev&searchon=names&suite=all&section=all. i think its packaging originally came from debian.

Reply:
+1 to this, but otherwise approved.

Reply:
@tchaikov I thought about that but the fmy module shows the commit we upgraded like so.

$ git tag --contains 51bf9cfacb644659e5d9c7e6fe66396726f2f4f4|head -1
7.0.0

I assumed we would want this commit in any version used right? Otherwise, what was the point of including it?

Reply:
actually there is no need to included fmt submodule if we just want to build on recent release of GNU/Linux mainstream distros.

there are two reasons why we need this submodule:

- bionic build. see https://packages.ubuntu.com/bionic/libfmt-dev . we had some test issues after migrating to ubuntu/focal. so we moved back to bionic. the root cause was unknown yet.
- 51bf9cfacb644659e5d9c7e6fe66396726f2f4f4 was included to address the build failure on MinGW.  i don't think MinGW is a "supported" distro as of now. i picked up the fmt change to unblock the community contributors who are interested in Windows port.

Reply:
So 5.3.0 or 6.0.0? You choose or should I flip a coin? :)

Reply:
i'd suggest 6.0.0 . as it's the version which deprecated `fmt/time.h`, and recommended `fmt/chrono.h`.